MetaMask/metamask-extension
# Restore support for Linea networks

## Explanation



When the NetworkController in this repo was replaced with `@metamask/network-controller`, support for Linea networks was lost as it did not support it at that time. `@metamask/network-controller` has since been updated, so this commit bumps that package to restore support.

Fixes https://github.com/MetaMask/metamask-extension/issues/19906.

## Screenshots/Screencaps



### Before



<img width="378" alt="Screenshot 2023-07-13 at 2 30 17 PM" src="https://github.com/MetaMask/metamask-extension/assets/7371/2cd2e507-ad52-48ef-a663-b0d9eeea1888">

<img width="1728" alt="Screenshot 2023-07-13 at 2 30 22 PM" src="https://github.com/MetaMask/metamask-extension/assets/7371/a4dfd1a1-e145-42ad-b474-48ea86a07e49">


### After



<img width="386" alt="Screenshot 2023-07-13 at 4 41 50 PM" src="https://github.com/MetaMask/metamask-extension/assets/7371/59d85785-838b-4899-8a31-9626a5422102">


## Manual Testing Steps



Switch to Linea Goerli. You should not see an error. (Alternatively, if you were already on Linea Goerli and you open the extension, you should see the home screen and MetaMask should not fail to start.)

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
